base.js

```diff
@@ -2,12 +2,11 @@ module.exports = {
   parserPreset: 'conventional-changelog-conventionalcommits',
   rules: {
     'body-leading-blank': [2, 'always'],
-    'body-max-line-length': [2, 'always', 72],
+    'body-max-line-length': [2, 'always', 100],
```

Comment:
Since the change is for dependabot and they go up to 80 can we instead make this 80?